crates/superposition/src/app_state.rs

```diff
@@ -124,6 +105,27 @@ pub async fn get(
         redis: redis_pool,
         http_client: reqwest::Client::new(),
         kms_client: kms_client.clone(),
-        tenant_configs,
+        encrypted_keys: join_all(get_from_env_unsafe::<String>("ENCRYPTED_KEYS")
+            .expect("ENCRYPTED_KEYS is not set")
```

Comment:
this should not be a mandatory key
```suggestion
        encrypted_keys: join_all(get_from_env_or_default::<String>("ENCRYPTED_KEYS", String::new())
```